docs/en_US/NAS/retiarii/Tutorial.rst

```diff
@@ -182,13 +182,19 @@ Here is a simple example of using trainer and strategy.
 
 .. code-block:: python
 
-  trainer = PyTorchImageClassificationTrainer(base_model,   
-    dataset_cls="MNIST",
-    dataset_kwargs={"root": "data/mnist", "download": True},
-    dataloader_kwargs={"batch_size": 32},
-    optimizer_kwargs={"lr": 1e-3},
-    trainer_kwargs={"max_epochs": 1})
-  simple_startegy = RandomStrategy()
+  import nni.retiarii.trainer.pytorch.lightning as pl
+  from nni.retiarii import blackbox
+  from torchvision import transforms
+
+  transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))])
+  train_dataset = blackbox(MNIST, root='data/mnist', train=True, download=True, transform=transform)
+  test_dataset = blackbox(MNIST, root='data/mnist', train=False, download=True, transform=transform)
+  lightning = pl.Lightning(pl.Classification(),
```

Comment:
`lightning` is not a good name, because it is not easy to know what it is. `classification_trainer` might be better